src/screens/king-of-the-hill/components/StepInidicators.tsx

```diff
@@ -0,0 +1,91 @@
+import React, { memo } from 'react';
```

Comment:
can you fix the name of this file itself - typo: `StepInidicators` instead of `StepIndicators` and corresponding places that import from here